lib/ASRouterTargeting.jsm

```diff
@@ -417,10 +450,13 @@ const TargetingGetters = {
     return null;
   },
   get isFxABadgeEnabled() {
-    return Services.prefs.getBoolPref(
-      "browser.messaging-system.fxatoolbarbadge.enabled",
-      false
-    );
+    return isFxABadgeEnabled;
+  },
+  get userPrefs() {
```

Comment:
This will need some refactoring after groups, but works for now. Do you want to also add the snippets pref in here?